src/components/PageHeader.tsx

```diff
@@ -52,11 +51,39 @@ export const PageHeader = ({ backPath, children, ...props }: PageHeaderProps) =>
   );
 };
 
+const StyledButton = styled(Button)`
+  background-color: ${({ theme }) => theme.palette.section.light};
+  color: ${({ theme }) => theme.palette.section.megaDark};
+  padding: 0.2rem;
+  min-width: 0;
+`;
+
 const StyledRegistrationPageHeader = styled(PageHeader)`
   font-weight: 700;
   font-style: italic;
 `;
 
-export const RegistrationPageHeader = (props: PageHeaderProps) => (
-  <StyledRegistrationPageHeader variant="h2" variantMapping={{ h2: 'h1' }} {...props} />
-);
+export const RegistrationPageHeader = (props: { children: string }) => {
+  const [showFullText, setShowFullText] = useState(false);
+
+  const toggleFullText = () => setShowFullText(!showFullText);
+
+  return (
+    <StyledRegistrationPageHeader variant="h2" variantMapping={{ h2: 'h1' }} onClick={toggleFullText} {...props}>
+      {showFullText ? (
+        props.children
+      ) : (
+        <TextTruncate
+          line={2}
+          truncateText="..."
+          text={props.children}
```

Comment:
hvis jeg virkelig skal pirke hadde det vært fint om tittelen fikk `cursor: pointer` hvis man kan toggle fullText

Reply:
åh, hadde jo egentlig det - sikkert "ryddet" litt for mye 😅